[
 {
  "author": "arfrever", 
  "date": "2009-04-04T19:19:56.471593Z", 
  "msg": [
   "Python 3 compatibility:", 
   "Don't use callable().", 
   "* subversion/tests/cmdline/svntest/testcase.py", 
   "(TestCase.__init__): Don't use callable()."
  ], 
  "revision_id": "877075"
 }, 
 {
  "author": "arfrever", 
  "date": "2009-04-04T19:17:31.667499Z", 
  "msg": [
   "On the 'python-3-compatibility' branch:", 
   "Fix sorting of instances of svntest.tree.SVNTreeNode.", 
   "* subversion/tests/cmdline/svntest/tree.py", 
   "(SVNTreeNode.__lt__ ,SVNTreeNode.__gt__): New functions."
  ], 
  "revision_id": "877074"
 }, 
 {
  "author": "arfrever", 
  "date": "2009-04-04T19:09:18.805219Z", 
  "msg": [
   "* subversion/tests/cmdline/commit_tests.py", 
   "(from_wc_top_with_bad_editor): Use list comprehension instead of map()."
  ], 
  "revision_id": "877073"
 }, 
 {
  "author": "arfrever", 
  "date": "2009-04-04T19:01:40.672093Z", 
  "msg": [
   "Python 3 compatibility:", 
   "Update calls to exec().", 
   "* subversion/tests/cmdline/svntest/main.py: Update calls to exec()."
  ], 
  "revision_id": "877072"
 }, 
 {
  "author": "arfrever", 
  "date": "2009-04-04T18:46:26.891893Z", 
  "msg": [
   "Follow-up to r33236:", 
   "* subversion/tests/cmdline/copy_tests.py", 
   "(change_case_of_hostname): Use str.upper() and str.lower() instead of", 
   "string.upper() and string.lower() for compatibility with Python 3."
  ], 
  "revision_id": "877071"
 }
]